### ISSUE TITLE
Documentation outdated - :allow_mass_assignment => true

### ISSUE THREAD

**Issue by brendon** [collectiveidea/audited#223]
It seems that the option `:allow_mass_assignment => true` isn't even referenced anymore in the code. There is a confusing bit of documentation in the README relating to Rails 3.x and `strong_parameters` which led me to retain the `:allow_mass_assignment` option until I did a bit more digging.

I'd be happy to put in a pull request to remove this documentation. Alternatively, if you're expecting `audited` to cope with the `protected_attributes` gem then perhaps this functionality should be added back in?

And if I've not been searching the code properly and this option does indeed have an effect, I'd love to know where in the code it is! :D

**Comment by Tonkpils** [comment]
Ditto on this, this cause me a bit of a headache. Why does 4.2 not play nicely with strong_params without adding the `allow_mass_assignment` option and why is that not used anywhere in the code? I couldn't find any reference to it.

**Comment by austenmadden** [comment]
I realize this is a almost two year old issue, but I'd be interested to know the answer.

**Comment by domcleal** [comment]
I don't think it has an effect any more, and the docs are therefore out of date. The option was removed in 9249219, so it is effectively now always `true`.

When using Rails 3 or Rails 4 with protected_attributes, the default behaviour of audited 3.x was to call `attr_accessible :audit_comment` which would automatically require whitelisting every other permitted attribute on the model with `attr_accessible`.

Using `allow_mass_assignment => true` would disable this, so you could use the older blacklist method of `attr_protected` or possibly strong parameters with `params.permit(..)` without interference.

TLDR: Since audited 4.x and its removal, it is now up to the user of the library to permit the `:audit_comment` attribute (if you're using it) by calling `attr_accessible` when using the protected_attributes gem on 4.x, or to add it to your `params.permit(..)` call on most modern RoR 4/5 apps.

**Comment by brendon** [comment]
Sounds good :)
